piptools/utils.py

```diff
@@ -522,3 +525,104 @@ def parse_requirements_from_wheel_metadata(
             markers=parts.markers,
             extras=parts.extras,
         )
+
+
+def pyproject_toml_defaults_cb(
+    ctx: click.Context, param: click.Parameter, value: str | None
+) -> str | None:
+    """
+    Defaults for `click.Command` parameters should be override-able in pyproject.toml
```

Comment:
Could you make the docstring title describe what the function does following PEP 257?

Reply:
Marking this thread as resolved.